assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -208,8 +208,8 @@ $z-layers: (
 		'.detail-page__action-buttons': 200
 	),
 	'.popover': (
-		'.input-chrono__container .gridicons-calendar': 0,
-		'input.input-chrono': 1,
+		'.input-chrono .gridicons-calendar': 0,
+		'input.input-chrono__input': 1,
```

Comment:
The `input.input-chrono__input` selector can be shortened to just `.input-chrono__input`. The extra specificity doesn't seem to be needed anywhere.

Reply:
That's a good point. 👍🏽 The extra specificity was present earlier as well, and I thought I might retain it. I have removed the extra specificity on c887e3632e3c75ba4084f4f48773a7cd78d38fed.